backends/jgroups/src/test/java/org/hibernate/search/backend/jgroups/impl/SyncJGroupsBackendTest.java

```diff
@@ -114,10 +114,15 @@ public void testSynchAsConfigured() {
 		}
 		catch (SearchException se) {
 			//Expected: we're inducing the RPC into NPE
-			Throwable cause = se.getCause().getCause().getCause();
-			Assert.assertTrue( "Cause was not a NullPointerException but a " + cause, cause instanceof NullPointerException );
-			Assert.assertEquals( "Simulated Failure", cause.getMessage() );
-			npeTriggered = true;
+			try {
```

Comment:
Well, it's related because I needed the exception to understand what was going on and fix it :)